staging/src/k8s.io/apiserver/pkg/server/genericapiserver_test.go

```diff
@@ -517,19 +517,24 @@ func TestGracefulShutdown(t *testing.T) {
 		return handler
 	}
 
-	handler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+	s, err := config.Complete(nil).New("test", NewEmptyDelegate())
+	if err != nil {
+		t.Fatalf("Error in bringing up the server: %v", err)
+	}
+
+	twoSecondHandler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		wg.Done()
 		time.Sleep(2 * time.Second)
 		w.WriteHeader(http.StatusOK)
 		graceShutdown = true
 	})
+	okHandler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		wg.Done()
```

Comment:
Do we need the call to Done here? We don't seem to call '/200' until after the wait.